<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登錄框</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::AddressBookNew"/>
  </property>
  <property name="windowOpacity">
   <double>0.950000000000000</double>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>小賴字體 等寬 SC</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>請輸入帳號</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>小賴字體 等寬 SC</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>帳號</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>小賴字體 等寬 SC</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>請輸入密碼</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>小賴字體 等寬 SC</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密碼</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>425</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>login</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
